src/components/Web3Status/index.tsx

```diff
@@ -214,7 +214,8 @@ function Web3StatusInner() {
 
   const hasPendingTransactions = !!pending.length
   const hasSocks = useHasSocks()
-  const toggleWallet = navbarFlag === NavBarVariant.Enabled ? toggleWalletDropdown : toggleWalletModal
+  const isNavbarEnabled = navbarFlag === NavBarVariant.Enabled
```

Comment:
You're only ever checking that it's enabled, so you can simplify this further, here and in other files:

    const isNavBarEnabled = useNavBarFlag() === NavBarVariant.Enabled